test/selenium/lib/view/log_in/accessors.js

```diff
@@ -0,0 +1,24 @@
+var By = require('selenium-webdriver').By;
+var Accessors = require('../accessors');
+
+
+function LoginAccessors() {
+  Accessors.apply(this, arguments);
+}
+
+LoginAccessors.prototype = Object.assign({
+
+  get passwordField() {
+    return this.waitForElement(By.id('signup-pwd1'));
+  },
```

Comment:
The problem I have is that using those selectors, and the way they are defined, they are not located. Remember what was necessary for the app, like ait for the element to be located and displayed...other way, seems like they are not found